shared/js/media/remote_controls.js

```diff
@@ -197,11 +180,6 @@ MediaRemoteControls.prototype._setupBluetooth = function(callback) {
       self._commandHandler(AVRCP.PLAY_PRESS);
     }
   }
-
-  function resetDefaultAdapter() {
-    self.defaultAdapter = null;
-    // Do we need to do anything else?
-  }
```

Comment:
@ian-liu do you think we need add callback to reset the adapter in bluetoothHelper?